Qast/ViewControllers/MapViewController.swift

```diff
@@ -14,26 +14,13 @@ import CoreGraphics
 
 class MapViewController: UIViewController {
 
-    var currentSoundZone: SoundZone? {
-        didSet {
-            if let currentSoundZone = currentSoundZone {
-                self.title = currentSoundZone.id
-                notificationManager.displaySoundZoneChangeNotification(currentSoundZone, spoken: true)
-                streamingManager.stop()
-                streamingManager.enqueue(currentSoundZone.streamId)
-            } else {
-                self.title = "Not in any SoundZone"
-                notificationManager.displaySoundZoneChangeNotification(currentSoundZone, spoken: true)
-                streamingManager.stop()
-            }
-        }
-    }
+    let locationManager: LocationManager = LocationManager()
     
     var isInitialLocationUpdate: Bool = true
     
     var nearbySoundZones: [SoundZone]?
     
-    let mapView = QastMapView()
+    var mapView: MGLMapView = QastMapView()
```

Comment:
unsure why, but adding `mapView` as a required property in the `LocationManagerDelegate` as a `var` was easier than as a `let`. Something about getters and setters.

Anyways, this could be changed to a let if I knew a bit more about protocol properties I'm sure.